lms/envs/docker-production.py

```diff
@@ -51,10 +51,11 @@ def get_docker_logger_config(log_dir='/var/tmp',
             },
             'tracking': {
                 'level': 'DEBUG',
-                'class': 'logging.handlers.SysLogHandler',
-                'address': ('localhost', 5140),
-                'facility': SysLogHandler.LOG_LOCAL1,
+                'class': 'logging.handlers.RotatingFileHandler',
+                'filename': '/var/tmp/tracking_logs.log',
+                'backupCount': 5,
```

Comment:
Are we at all concerned about 5 log files eating up disk space?

Reply:
10485760 bytes -> 10 MB, so 5 is 50 MB. Is that too much?